### PR TITLE
HW-53397: No search bar appears on Select Country screen for iPhone 5 iOS 10

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**
Reverted tableView.style to .plain to make SearchBar stick to the screen top

### CODE REVIEW

**Comment by oleksii-lubianyi-epam** [comment]
With tableView.style == .plain search bar is stick to the table top.
But in this case we have returning issue of bottom separator - it’s with inset from left, but it should not (see video in attached .zip)
[searchBar.mov.zip](https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/files/3262499/searchBar.mov.zip)

To fix this we may pass to cell `isLast` parameter and set `inset.left = isLast ? 0 : <theme value>`
I don’t love this solution, though

File: Sources/Generics/GenericTableViewController.swift [hunk 1]
```diff
@@ -205,7 +205,7 @@ private extension GenericTableViewController {
     }
 
     func setupTable() {
-        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView = UITableView(frame: .zero, style: .plain)
```
Comment (jsaini-hw) [inline Sources/Generics/GenericTableViewController.swift]: update this to 
``` if #available(iOS 11.0, *) {
            tableView = UITableView(frame: .zero, style: .grouped)
            tableView.tableFooterView = UIView()
        } else {
            tableView = UITableView(frame: .zero, style: .plain)
            let footerView = UIView(frame: CGRect(x: 0, y: 0, width: tableView.frame.size.width, height: 0.5))
            footerView.backgroundColor = tableView.separatorColor
            tableView.tableFooterView = footerView
        }
```

Comment (syyang-hw) [reply Sources/Generics/GenericTableViewController.swift]: @jsaini-hw is right. We already had similar workaround in SelectTransferMethodTypeTableViewController.swift, setupTransferMethodTypeTableView()

Comment (oleksii-lubianyi-epam) [reply Sources/Generics/GenericTableViewController.swift]: @jsaini-hw, @syyang-hw 
Thanks, changed.

Btw, in my branch at least `setupTransferMethodTypeTableView()` doesn't have this workaround.
Should I add it there as well?

**Pull request merged**